tox.ini

```diff
@@ -3,12 +3,15 @@ envlist = py35-django22-stretch,py38-django22-buster
 
 
 [testenv]
+passenv = CI CIRCLECI CIRCLE_* CODECOV_TOKEN
 envdir=
     # Use the same environment for all commands running under a specific python version
     py35: {toxworkdir}/py35
     py38: {toxworkdir}/py38
 
 deps =
     -rrequirements/test.txt
+    codecov
```

Comment:
What do you think if we add codecov to test.in or other of these files? So we are using always the same version and avoid problems if a new version of codecov has  unexpected changes

Reply:
no problem